submissions/NadiaVorontsova/classic-frogger-game/app.js

```diff
@@ -0,0 +1,136 @@
+const startPlayerPositionX = 200,
+    startPlayerPositionY = 390,
+    cellWidth = 101,
+    cellHeight = 83,
+    startX = 0,
+    startY = 0,
+    rightWall = 400,
+    bottomWall = 350;
```

Comment:
By convention the global constants names of this kind are ALL_CAPITALS. This way when we see a variable all capitals we instantly understand it is a global constant.

Reply:
Changed to capital letters